devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/singleton-examples/google-cloud-functions-example/codestart.yml

```diff
@@ -0,0 +1,10 @@
+name: google-cloud-functions-example
+ref: google-cloud-functions
```

Comment:
Sorry I missed that you didn't add the extension binding in the descriptor metadata:
https://github.com/quarkusio/quarkus/blob/master/extensions/google-cloud-functions/runtime/src/main/resources/META-INF/quarkus-extension.yaml
```yaml
codestart: google-cloud-functions
```

(This is what trigger the codestart activation when the extension is selected)

I am not sure if the binding should be on `google-cloud-functions` or `google-cloud-functions-http` (or both)

Reply:
It's `google-cloud-functions` only, I'll provide a separate PR for HTTP binding and for funqy.
I added it in the corresponing `quarkus-extension.yaml` file.

By the way, this step is missing from the documentation: https://github.com/quarkusio/quarkus/tree/master/independent-projects/tools/codestarts

Reply:
Weird the adoc rendering is not working, it was working fine a few weeks ago..

@loicmathieu it is :) look at the "When done (to bind the extension codestart to the actual extension)" section

Reply:
Ah OK, I read too quickly.

This deserve a little code snippet from an extension :)